### PR TITLE
Teleop switch

### PR DESCRIPTION
**Pull request by CatonQ**

### CODE REVIEW

File: src/main/java/frc/robot/SetupShuffleboard.java [hunk 1]
```diff
@@ -119,15 +121,17 @@ public static void setupShuffleboard(SwerveSubsystem swerve, PizzaBoxSubsystem p
...
+    Shuffleboard.getTab("Climb").add("unclimb2", new UnClimbPartTwoThatWillBringDownTheMotor(climbSubsystem, swerve, armSubsystem, reactionSubsystem)).withPosition(5, 1);
 
     Shuffleboard.getTab("Climb").add("STOP CLIMB!!!!", new StopClimb(climbSubsystem)).withSize(2, 1).withPosition(2, 2);
 
     Shuffleboard.getTab("Climb").addDouble("Reaction Bar Angle", ()-> reactionSubsystem.getPos()).withPosition(9, 4);
 
 
     Shuffleboard.getTab("System Check").add("check", new SystemCheck(armSubsystem, climbSubsystem, intakeSubsystem, pizzaBoxSubsystem, reactionSubsystem, swerve));
+  
+    Shuffleboard.getTab("Arm").addDouble("distance", ()-> UtilMath.distanceFromBlueSpeaker(swerve.getPose()));
```
Comment (isaacngkh) [inline src/main/java/frc/robot/SetupShuffleboard.java]: Can we rename this to "distance from blue speaker"?

We should also upgrade the method to do distanceFromSpeaker and just check which speaker dynamically

File: src/main/java/frc/robot/commands/ClimberCommands/ClimbParts/ClimbAndShoot.java [hunk 2]
```diff
@@ -26,10 +26,10 @@ public ClimbAndShoot(Climbsubsystem c, SwerveSubsystem s, ArmSubsystem a, PizzaB
     // Add your commands in the addCommands() call, e.g.
     addCommands (
       new PrintCommand("climb and shoot initialize"),
-      // new SpinToArmAngle(a, 135).withTimeout(1),
-      // Commands.waitUntil(()->a.checkEncoderAngleForClimb()),
-      new MotorDown(c).withTimeout(3).alongWith(new Extend(r)),
-      // new ScoreInTrapStutter(p, a),
+      new SpinToArmAngle(a, 260).withTimeout(1),
+      Commands.waitUntil(()->a.checkEncoderAngleForClimb()),
+      new MotorDown(c).alongWith(new Extend(r).withTimeout(3)),
```
Comment (isaacngkh) [inline src/main/java/frc/robot/commands/ClimberCommands/ClimbParts/ClimbAndShoot.java]: parenthesis in the wrong place
new MotorDown(c).alongWith(new Extend(r)).withTimeout(3),

File: src/main/java/frc/robot/commands/swervedrive/drivebase/TeleopDrive.java [hunk 3]
```diff
@@ -68,9 +69,24 @@ public void initialize()
...
-    double yVelocity   = Math.pow(vY.getAsDouble(), 3);
-    double angVelocity = Math.pow(omega.getAsDouble(), 3);
+     double angVelocity;
+     double xVelocity;
+     double yVelocity;
+    if(DriverStation.getAlliance().isPresent() && DriverStation.getAlliance().get() == DriverStation.Alliance.Blue)
+{
+     xVelocity   = Math.pow(vX.getAsDouble(), 3);
+     yVelocity   = Math.pow(vY.getAsDouble(), 3);
+     angVelocity = Math.pow(omega.getAsDouble(), 3);
```
Comment (isaacngkh) [inline src/main/java/frc/robot/commands/swervedrive/drivebase/TeleopDrive.java]: Can we fix the tabs here

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 4]
```diff
@@ -45,7 +45,7 @@ public static final class Arm {
     public static final int TRAP_ANGLE = 290;
     public static final int SPEAKER_LOW_ANGLE = 165;
     public static final int SPEAKER_HIGH_ANGLE = 238;
-    public static final int INTAKE_ANGLE = 66;
+    public static final int INTAKE_ANGLE = 60;
```
Comment (isaacngkh) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java]: double check arm intake angle

File: src/main/java/frc/robot/commands/swervedrive/drivebase/TeleopDrive.java [hunk 5]
```diff
@@ -68,9 +69,24 @@ public void initialize()
   @Override
   public void execute()
   {
-    double xVelocity   = Math.pow(vX.getAsDouble(), 3);
-    double yVelocity   = Math.pow(vY.getAsDouble(), 3);
-    double angVelocity = Math.pow(omega.getAsDouble(), 3);
+     double angVelocity;
+     double xVelocity;
+     double yVelocity;
+    if(DriverStation.getAlliance().isPresent() && DriverStation.getAlliance().get() == DriverStation.Alliance.Blue)
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/swervedrive/drivebase/TeleopDrive.java]: Use a value that is -1 or +1 based on the logic, and multiply the x and y velocity.  Then you don't need to duplicate the logic.

File: src/main/java/frc/robot/subsystems/swervedrive/SwerveSubsystem.java [hunk 6]
```diff
@@ -569,4 +554,9 @@ public Field2d getField2d()
     return swerveDrive.field;
   }
 
+  public void setHeading()
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/swervedrive/SwerveSubsystem.java:566]: Not used - was this used for something else?

**Review by RobertTheBlair — APPROVED** [review]
Works, Nice to Have some cleanups

**Pull request merged**